app/views/external_users/claims/supporting_documents/_summary.html.haml

```diff
@@ -2,50 +2,50 @@
   %h2.govuk-heading-m
     Evidence supplied on disk
 
-  %table.summary
-    %caption.govuk-visually-hidden
+  = govuk_table do
+    = govuk_table_caption(class: 'govuk-visually-hidden') do
       = "Evidence supplied on disk"
-    %tbody
-      %tr
-        %th{ scope:'row', class: 'bold' }
+    = govuk_table_tbody do
+      = govuk_table_row do
+        = govuk_table_th(scope:'row', class: 'bold') do
```

Comment:
Class `.bold` is no longer required, as the `th` element now has the relevant font weight